client-core/src/main/java/com/influxdb/internal/AbstractQueryApi.java

```diff
@@ -58,12 +60,9 @@
 
     };
 
-    protected static final JSONObject DEFAULT_DIALECT = new JSONObject()
-            .put("header", true)
-            .put("delimiter", ",")
-            .put("quoteChar", "\"")
-            .put("commentPrefix", "#")
-            .put("annotations", new JSONArray().put("datatype").put("group").put("default"));
+    protected static final JsonObject DEFAULT_DIALECT = new JsonParser().parse("{\"header\":true,"
+        + "\"delimiter\":\",\",\"quoteChar\":\"\\\"\",\"commentPrefix\":\"#\","
+        + "\"annotations\":[\"datatype\",\"group\",\"default\"]}").getAsJsonObject();
```

Comment:
We could initialise `DEFAULT_DIALECT` as a `String` to avoid unnecessary `toString()` in usages .

```suggestion
    protected static final String DEFAULT_DIALECT;
    static {
        Map<String, Object> dialect = new HashMap<>();
        dialect.put("header", true);
        dialect.put("delimiter", ",");
        dialect.put("quoteChar", "\"");
        dialect.put("commentPrefix", "#");
        dialect.put("annotations", new String[]{"datatype", "group", "default"});
        DEFAULT_DIALECT = new GsonBuilder().create().toJson(dialect);
    }
```